src/api-utils/schemas/gameDev/achievementsSchemas.ts

```diff
@@ -33,7 +33,13 @@ export const postAchievementResponseSchema = Joi.object({
 
 export const postAchievementProgressRequestSchema = Joi.object({
   progress: Joi.number().optional(),
-}).required();
+  _user: Joi.object().optional(),
+});
```

Comment:
don't forget to remove the _user and add back the required, please

Reply:
Oh alright :)